src/components/cursor-controller.js

```diff
@@ -367,25 +341,13 @@ AFRAME.registerComponent("cursor-controller", {
   _updateController: function() {
     this.hasPointingDevice = this.controllerQueue.length > 0 && this.inVR;
 
-    this.el.setAttribute("line", { visible: this.hasPointingDevice });
+    this._setCursorVisibility(this.hasPointingDevice);
```

Comment:
This will hide the cursor outright in cases where you probably want to just hide the line. Maybe you want something like?
```
// refresh visibility state
this._setCursorVisibility(this.cursor.getAttribute("visible"))
```